src/games/tic_tac_toe.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+exports.setUp = function setUp(players) {
```

Comment:
Mayve check this in "functional style" using lodash/underscore or libraries like those two?